### PR TITLE
PEP8 fixes

### PR DESCRIPTION
**Pull request by bsipocz**
I'm sorry about this massive change, but allowing the pep8 test to fail results in a lots of diversion. Ultimately I plan to set up a bot that checks every PR and comments back nicely, but it's best if we do a big cleanup before setting that up.

There are still ~300 error/warning around, these we the most obvious that could be dealt with semi automatically.

E111,E124,E201,E202,E203,E211,E221,E225,E231,E241,E251,E261,E265,E271,E272,E301,E302,E303,E305,E502,E703,E711,E712,E714,W291,W292,W293,W391

### CODE REVIEW

File: astroquery/utils/tap/model/tests/test_job.py [hunk 1]
```diff
@@ -26,13 +26,13 @@ def data_path(filename):
...
     return os.path.join(data_dir, filename)
 
-class TestJob(unittest.TestCase):
 
+class TestJob(unittest.TestCase):
 
     def test_job_basic(self):
         job = Job(async=False)
         res = job.is_sync()
-        assert res == True, \
```
Comment (keflavich) [inline astroquery/utils/tap/model/tests/test_job.py]: Is there any chance of this causing a problem?  i.e., is there any chance that `res==2`, which would now evaluate to `True` but previously would not?

**Comment by keflavich** [comment]
Broadly fine.  I have some of these turned off locally but I don't see any problems with what has been corrected here.

File: astroquery/utils/tap/model/tests/test_job.py (hunk 1 above)
Comment (bsipocz) [inline astroquery/utils/tap/model/tests/test_job.py]: This should be fine, as it should always come from this as a bool:
```
    def is_sync(self):
        """Returns True if this job was executed synchronously

        Returns
        -------
        'True' if the job is synchronous
        """
        return not self.__async
```

Comment (bsipocz) [inline astroquery/utils/tap/model/tests/test_job.py]: also I think this is a bit superfluous to have one liners like this, but that's another story

**Comment by bsipocz** [comment]
I'm not sure why the coverage drops (same amount as in #859), I'm going ahead and ignore it here, too.

**Pull request merged**